src/dtclient/image.go

```diff
@@ -13,6 +15,23 @@ type LatestImageInfo struct {
 	Tag    string `json:"tag"`
 }
 
+func ImageInfoFromUri(imageUri string) *LatestImageInfo {
```

Comment:
We could use https://github.com/containers/image/blob/c1cc1eab4805282ce634c79c1c5fc424475647f8/docker/reference/reference.go#L189 the Parse() here and check what type of image has been passed